c/pthread-driver-races/char_pc8736x_gpio_pc8736x_gpio_open_pc8736x_gpio_configure.i

```diff
@@ -6632,7 +6632,10 @@ void cpu_relax() {
   return;
 }
 void *external_alloc(unsigned int size);
-void assume_abort_if_not(int);
+void abort(void);
+void assume_abort_if_not(int cond) {
+  if(!cond) {abort();}
+}
```

Comment:
Actually the definition is already present in this file in line 158:

https://github.com/sosy-lab/sv-benchmarks/blob/4014986c335d858a7a121bd75ef348fb398cedf3/c/pthread-driver-races/char_pc8736x_gpio_pc8736x_gpio_open_pc8736x_gpio_configure.i#L158-L161

Are you sure these changes are necessary?